### PR TITLE
[UXIT-1391] Fix link checker test

### PR DESCRIPTION
**Pull request by barbaraperic**
## Note
- added AIRTABLE env
- removed broken links 
- added condition to skip links with 999 status

🚨 **all_ecosystem_explorer_projects spec is still broken - check again after https://github.com/FilecoinFoundationWeb/filecoin-foundation/pull/460 gets merged** 🚨

### CODE REVIEW

File: cypress/support/verifyLinksUtil.ts [hunk 1]
```diff
@@ -14,7 +14,9 @@ export function verifyLinks(path: string) {
             let errorMessage = response
               ? `Status code: ${response.status}`
               : 'No response received or request timed out'
-            cy.log(`Broken link: ${href} - ${errorMessage}`)
+            if (response && response.status !== 999) {
```
Comment (mirhamasala) [inline cypress/support/verifyLinksUtil.ts:17]: What is a 999?

File: src/content/blog/filecoin-ecosystem-shines-at-fil-hong-kong-ai-depin-fvm-and-more.md [hunk 2]
```diff
@@ -28,7 +28,7 @@ seo:
 
 The Filecoin ecosystem was on full display at FIL Hong Kong, hosted by ND Labs and supported by Filecoin Foundation.
 
-The Filecoin ecosystem was on full display at [FIL Hong Kong](http://fil-hk.io), hosted by ND Labs and supported by Filecoin Foundation. Hundreds of attendees from around the world gathered in the burgeoning Web3 hub of Hong Kong to explore the latest developments in the Filecoin ecosystem and envision the future of decentralized storage.
+The Filecoin ecosystem was on full display at FIL Hong Kong, hosted by ND Labs and supported by Filecoin Foundation. Hundreds of attendees from around the world gathered in the burgeoning Web3 hub of Hong Kong to explore the latest developments in the Filecoin ecosystem and envision the future of decentralized storage.
```
Comment (mirhamasala) [inline src/content/blog/filecoin-ecosystem-shines-at-fil-hong-kong-ai-depin-fvm-and-more.md:31]: Please use the appropriate event link from fil.org/events

File: src/content/blog/fresh-from-ff-early-march-2024.md [hunk 3]
```diff
@@ -68,16 +68,16 @@ _Check out the latest updates about what the Filecoin Foundation team has been u
 
 **Marketing & Comms**
 
-- Catch FF’s Clara Tsao talking about [Filecoin & AI: Building the Next Generation of Data Storage](https://www.ethdenver.com/agenda/filecoin-ai-building-the-next-generation-of-data-storage-ad010), Friday, March 1, on the Neptune Stage at ETHDenver.
+- Catch FF’s Clara Tsao talking about Filecoin & AI: Building the Next Generation of Data Storage, Friday, March 1, on the Neptune Stage at ETHDenver.
```
Comment (mirhamasala) [inline src/content/blog/fresh-from-ff-early-march-2024.md:71]: Please use the appropriate event link from fil.org/events

File: src/content/blog/fresh-from-ff-early-march-2024.md [hunk 4]
```diff
@@ -68,16 +68,16 @@ _Check out the latest updates about what the Filecoin Foundation team has been u
...
 
 **Events**
 
 - Check out the [Filecoin at ETHDenver website](https://hub.fil.org/ethdenver2024) for all of the Filecoin community events happening throughout the week.
-- The Filecoin Orbit team announced an ETHDenver [bounty](https://www.ethdenver.com/buidl-sponsors/filecoin-foundation) to build an app to track and reward the activities of the Filecoin Orbit ambassador program in the form of NFT “achievements” and associated ERC20 token reward points. The aim of the bounty is to build an app that creates a standard rewards system for the Orbit program on FVM.
+- The Filecoin Orbit team announced an ETHDenver bounty to build an app to track and reward the activities of the Filecoin Orbit ambassador program in the form of NFT “achievements” and associated ERC20 token reward points. The aim of the bounty is to build an app that creates a standard rewards system for the Orbit program on FVM.
 - Don’t miss our upcoming community events!
 
-  - 4/5-4/8 - [FIL Hong Kong](https://fil-hk.io/) hosted by NDLabs alongside Hong Kong Web3 Festival
+  - 4/5-4/8 - FIL Hong Kong hosted by NDLabs alongside Hong Kong Web3 Festival
```
Comment (mirhamasala) [inline src/content/blog/fresh-from-ff-early-march-2024.md:80]: Please use the appropriate event link from fil.org/events

File: src/content/blog/fresh-from-ff-mid-february-2024.md [hunk 5]
```diff
@@ -39,7 +39,7 @@ Connect with the [Filecoin community at ETHDenver](https://hub.fil.org/events/et
   1. The [Data Economy Hackathon](https://dorahacks.io/hackathon/filecoin-data-economy/detail) is a six-week virtual hackathon beginning on February 18 and focusing on Filecoin and IPC. The hackathon emphasizes projects that harness data value, innovate for scalability, create dApps utilizing Filecoin, and enhance the network.
   2. The [Filecoin track at ETHGlobal London](https://ethglobal.com/events/london2024/prizes/filecoin), March 15-17, will be an in-person Filecoin hackathon — track details will be announced soon.
 
-- FF’s developer advocacy team will be on the ground at [ETHDenver](https://hub.fil.org/events/ethdenver/see-the-schedule) and [FIL Hong Kong](https://fil-hk.io/). Reach out if you want to set up time to chat with developer advocates at either event.
+- FF’s developer advocacy team will be on the ground at [ETHDenver](https://hub.fil.org/events/ethdenver/see-the-schedule) and FIL Hong Kong. Reach out if you want to set up time to chat with developer advocates at either event.
```
Comment (mirhamasala) [inline src/content/blog/fresh-from-ff-mid-february-2024.md:42]: Please use the appropriate event link from fil.org/events

File: src/content/blog/fresh-from-ff-mid-march-2024.md [hunk 6]
```diff
@@ -67,4 +67,4 @@ _Check out the latest updates about what the Filecoin Foundation team has been u
 
 **Events**
 
-- Join us April 5-8 at [FIL Hong Kong](https://fil-hk.io/), hosted by ND Labs, alongside the Hong Kong Web3 Festival, for storage provider-focused lightning talks, workshops, and plenty of coworking space for those in the Filecoin ecosystem! If you’re a Filecoin storage provider or want to explore decentralized storage for your business, join us for exciting talks focused on Web3 trends, decentralized storage, DePIN, and more.
+- Join us April 5-8 at FIL Hong Kong, hosted by ND Labs, alongside the Hong Kong Web3 Festival, for storage provider-focused lightning talks, workshops, and plenty of coworking space for those in the Filecoin ecosystem! If you’re a Filecoin storage provider or want to explore decentralized storage for your business, join us for exciting talks focused on Web3 trends, decentralized storage, DePIN, and more.
```
Comment (mirhamasala) [inline src/content/blog/fresh-from-ff-mid-march-2024.md:70]: Please use the appropriate event link from fil.org/events

File: src/content/blog/help-us-improve-filecoin-docs-plus-how-to-receive-grant-funding-up-to-50-000.md [hunk 7]
```diff
@@ -58,7 +58,8 @@ In 2024, the FF Grants Program is looking for proposals that address:
 Grant graduates have gone on to make a huge impact in the Filecoin ecosystem. You can look at recent grant recipients in [FF’s 2023 Annual Report](https://www.figma.com/proto/OpRYFQO34ycc5N7YIMeGAe/2023-FF-Annual-Report?node-id=1-11370). Here are a few highlights:
 
 - [Portrait](https://portrait.gg/): An open-source web page builder that lets you turn your blockchain address into a website. Portrait web pages are automatically pinned to IPFS and stored on Filecoin, making the sites secure and resilient by design. Portrait’s goal is to enable communities to develop creative work in a decentralized way –– from custom blogs for writers, to online portfolios for designers, to landing pages for podcast hosts.
-- [Cavalry](https://cavalry.scenegroup.co/): 2D animation software for Mac and Windows. Users can create generative, abstract artwork for dynamic rendering in Cavalry, and automatically upload all generated artworks to NFT.storage, which provides storage using IPFS and Filecoin.
+  // UNDO
```
Comment (mirhamasala) [inline src/content/blog/help-us-improve-filecoin-docs-plus-how-to-receive-grant-funding-up-to-50-000.md:61]: Leftover?

Comment (mirhamasala) [reply src/content/blog/help-us-improve-filecoin-docs-plus-how-to-receive-grant-funding-up-to-50-000.md:61]: Link below works: https://cavalry.scenegroup.co/

File: src/content/blog/introducing-the-future-rules-a-new-podcast-series-from-the-filecoin-foundation-and-forkast-news.md [hunk 8]
```diff
@@ -25,6 +25,6 @@ Today, we’re thrilled to announce the launch of “[The Future Rules](https://
...
+Our [debut episode](https://the-future-rules-forkast-news-x-filecoin-foundation.simplecast.com/episodes/decentralized-finance) takes on decentralized finance (DeFi) with guest Sheila Warren, head of Data, Blockchain and Digital Assets at the World Economic Forum, where she also serves as a member of the Executive Committee. Sheila kicks off by explaining what DeFi is, and then takes a look at the potential it offers for financial inclusion before delving into the rumblings of a showdown between decentralized finance and regulators. She also explains how the WEF’s Policy-Maker Toolkit can help policymakers negotiate that tightrope and why we are all better off if “premature regulation” is avoided.
```
Comment (mirhamasala) [inline src/content/blog/introducing-the-future-rules-a-new-podcast-series-from-the-filecoin-foundation-and-forkast-news.md:28]:
```suggestion
Our [debut episode](https://the-future-rules-forkast-news-x-filecoin-foundation.simplecast.com/episodes/decentralized-finance) takes on decentralized finance (DeFi) with guest Sheila Warren, head of Data, Blockchain and Digital Assets at the World Economic Forum, where she also serves as a member of the Executive Committee. Sheila kicks off by explaining what DeFi is, and then takes a look at the potential it offers for financial inclusion before delving into the rumblings of a showdown between decentralized finance and regulators. She also explains how the WEF’s Policy-Maker Toolkit (link previously available but now unavailable) can help policymakers negotiate that tightrope and why we are all better off if “premature regulation” is avoided.
```

**Review by mirhamasala — CHANGES_REQUESTED** [review]
@barbaraperic Please check the comments. 🙏🏼

File: cypress/support/verifyLinksUtil.ts (hunk 1 above)
Comment (barbaraperic) [reply cypress/support/verifyLinksUtil.ts:17]: Request Denied - we get it for the FF LinkedIn page

Comment (mirhamasala) [reply cypress/support/verifyLinksUtil.ts:17]: Got it! I'm pushing a CR to abstract magic numbers like these and to refactor the code a bit for clarity and logic.

**Review by mirhamasala — APPROVED** [review]
Please have a look at the CR [>>](https://github.com/FilecoinFoundationWeb/filecoin-foundation/pull/459/commits/cd98d304819dacea2552426438b82c3fc975b160) - It's a tiny refactor. If you agree with it, we can get this merged. Please feel free to change anything that you don't like.

File: cypress/support/verifyLinksUtil.ts (hunk 1 above)
Comment (barbaraperic) [reply cypress/support/verifyLinksUtil.ts]: Looks good, tho if it's ok with you I'd remove the logs of "Ignored links" - not the information we need? We just don't want to include them as broken links.

```
function isClientOrServerError(status: number) {
  return (
    status >= CLIENT_ERROR_THRESHOLD && status !== REQUEST_DENIED_STATUS_CODE
  )
}
```

**Pull request merged**